x-pack/solutions/observability/plugins/profiling/test/scout/api/tests/has_setup.spec.ts

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { expect, apiTest, test } from '@kbn/scout-oblt';
+import type { ScoutTestConfig } from '@kbn/scout-oblt';
+import { format as formatUrl } from 'url';
+import supertest from 'supertest';
+import { getRoutePaths } from '../../../../common';
+import { getProfilingPackagePolicyIds, deletePackagePolicy } from '../../common/utils/fleet';
+import {
+  cleanUpProfilingData,
+  loadProfilingData,
+  setupProfiling,
+} from '../../common/utils/profiling_data';
+
+const profilingRoutePaths = getRoutePaths();
+
+function getSupertest(config: ScoutTestConfig) {
+  const kibanaServer = config.metadata.config.servers.kibana;
+  const url = formatUrl({
+    ...kibanaServer,
+    auth: 'elastic:changeme',
+  });
+  return supertest(url);
+}
+
+apiTest.describe('Profiling is not setup and no data is loaded', { tag: ['@svlOblt'] }, () => {
+  apiTest.beforeAll(async ({ esClient, log, config }) => {
+    await cleanUpProfilingData({
+      es: esClient,
+      st: getSupertest(config),
+      logger: log,
+    });
+  });
+
+  test('Admin user', async ({ profilingClient }) => {
+    const adminRes = await profilingClient.adminUser({
+      endpoint: `GET ${profilingRoutePaths.HasSetupESResources}`,
+    });
+
+    const adminStatus = adminRes.body;
+    expect(adminStatus.has_setup).toBe(false);
```

Comment:
Not sure if it is available, but this would look better:
```suggestion
    expect(adminStatus.has_setup).toBeFalsy()
```